src/Command/MigrateCommand.php

```diff
@@ -22,10 +22,80 @@
 
 class MigrateCommand extends Command
 {
+    const HELP = <<< EOH
+Migrate a project or library to target Laminas, Expressive, and/or Apigility
+packages.
+
+<info>Basic Usage</info>
+<info>-----------</info>
+
+In most cases, the command can be run without any arguments, in which case it
+will migrate the project in the current directory, rewriting any files that
+contain references to Zend Framework artifacts to instead reference the Laminas
+equivalents.
+
+If you wish to specify a path other than the current working directory, use the
+--path option.
+
+<info>Excluding Files</info>
+<info>---------------</info>
+
+If you wish to exclude all files under a given directory from migration, use
+the --exclude (or -e) option. A common use case for that is "--exclude data"
+or "--exclude data/cache". The --exclude option can be issued multiple times,
+one for each directory you wish to exclude.
+
+To exclude individual files, use the --exclude-file (-x) option. This option
```

Comment:
I'd prefer to have just `--exclude` which will accept either directories and files, so we can do:
```--exclude data/cache --exclude dump.sql```

it will simplify user interface.

Reply:
I agree, but there are some logistical issues I need to solve before I can say one way or the other if it's possible. I'll attempt those shortly.

Reply:
Figured out a solution!